spec/models/book_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
+require 'rails_helper'
+require 'faker'
+
+describe Book, type: :model do
+  it do
+    should validate_presence_of(:title)
+  end
+  it do
```

Comment:
Always leave a new line between 'it' blocks

Reply:
Done